mlflow/tensorflow/__init__.py

```diff
@@ -627,6 +629,8 @@ def autolog(
     disable_for_unsupported_versions=False,
     silent=False,
     registered_model_name=None,
+    log_input_examples=True,
```

Comment:
@bali0019 @BenWilson2 This should default to `False` because input examples may contain sensitive data, and we want users to have to opt-in. Other autologging integrations default to `False` for this param.